src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const resultRobot = {};
+  let resultValue;
```

Comment:
Doesn't make sense to create this variable before loop, as you don't use it after